packages/core-contracts/contracts/proxy/OwnableUUPSImplementation.sol

```diff
@@ -0,0 +1,68 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./UniversalProxyImplementation.sol";
+import "../ownership/Ownable.sol";
+
+contract OwnableUUPSImplementation is UniversalProxyImplementation, Ownable {
+    struct UnstructuredProxyStorage {
+        address implementation;
+        bool simulatingUpgrade;
+    }
+
+    struct OwnableStorage {
```

Comment:
Ownable should not be implemented at this level. For example, SNXToken.sol inherits UUPSImplementation and Ownable.

Reply:
Yeah, that was my question on the call. if we needed it here or not